### PR TITLE
Add bisect to talking points

### PR DESCRIPTION
**Pull request by feature-engineer**
People learning python should be familiar with python's standard library, this functionality is already included in python in the form of bisect.bisect_left with minimal wrapping for non existing values.

### CODE REVIEW

**Comment by feature-engineer** [comment]
@cmccandless That is why it is in talking points, and not in acceptable solutions. It's not a solution to this problem, but should be mentioned, as it is the correct and better solution to this problem outside the exercise.

File: tracks/python/exercises/binary-search/mentoring.md [hunk 1]
```diff
@@ -67,4 +67,18 @@ def binary_search(list_of_numbers, number):
...
+import bisect
+
+
+def binary_search(list_of_numbers, number):
+    idx = bisect.bisect_left(list_of_numbers, number)
+    if idx == len(list_of_numbers):
+        raise ValueError("number not found")
+    if list_of_numbers[idx] == number:
+        return idx
+    raise ValueError("number not found")
```
Comment (cmccandless) [inline tracks/python/exercises/binary-search/mentoring.md]: Why not follow the example in the [docs](https://docs.python.org/3/library/bisect.html)?
```
if idx != len(list_of_numbers) and list_of_numbers[idx] == number:
	return idx
raise ValueError("number not found")
```

Comment (feature-engineer) [reply tracks/python/exercises/binary-search/mentoring.md]: That's a good suggestion, done.

**Comment by SleeplessByte** [comment]
Thank you @cmccandless for the review and vigilance. @feature-engineer great addition!

**Pull request merged**